<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>championatWidget</class>
 <widget class="QWidget" name="championatWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Запись следующих ставок</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="CTableViewWd" name="commonView"/>
   </item>
   <item>
    <widget class="CTableViewWd" name="resultView">
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CTableViewWd</class>
   <extends>QTableView</extends>
   <header>table_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
